pydocx/export/numbering_span.py

```diff
@@ -206,6 +208,103 @@ def get_numbering_level(self, paragraph):
             return None
         return level
 
+    def detect_parent_child_map_for_items(self):
```

Comment:
I just realized that this methods needs to be adapted to handle more cases, especially any nested separated list. Will look into it tomorrow and try to adapt for all cases. Any feedback is welcome.